manifest.yml

```diff
@@ -1,6 +1,7 @@
 ---
 applications:
   - name: opre-ops-test
+    memory: 700M
```

Comment:
[todo] Can you include the reasons why we need this in the PR description?

Reply:
Yes, great idea, thank you